backend/app/Models/Role.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Role extends Model
+{
+    use HasFactory, SoftDeletes;
```

Comment:
add protected guarded

```suggestion
    use HasFactory, SoftDeletes;
    
    protected $guarded = [];
```